roachtest: sqlsmith/setup=empty/setting=no-mutations failed

roachtest.sqlsmith/setup=empty/setting=no-mutations [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6107800?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6107800?buildTab=artifacts#/sqlsmith/setup=empty/setting=no-mutations) on master @ [f4042d47fa8062a612c38d4696eb6bee9cee7c21](https://github.com/cockroachdb/cockroach/commits/f4042d47fa8062a612c38d4696eb6bee9cee7c21):


```
test artifacts and logs in: /artifacts/sqlsmith/setup=empty/setting=no-mutations/run_1
	sqlsmith.go:265,sqlsmith.go:325,test_runner.go:896: error: pq: internal error: crdb_internal.set_compaction_concurrency(): compaction_concurrency must be > 0
		stmt:
		SELECT
			tab_113526.col_194263 AS col_194265, 453213032:::INT8 AS col_194266
		FROM
			(
				VALUES
					(true),
					(true),
					(NULL),
					(false),
					(
						crdb_internal.set_compaction_concurrency(127:::INT8::INT8, (-6686260172759532247):::INT8::INT8, (-8623930486684336554):::INT8::INT8)::BOOL
					)
			)
				AS tab_113526 (col_194263),
			(VALUES ((-252947180):::INT8), (79985758:::INT8), ((-964813711):::INT8)) AS tab_113527 (col_194264)
		WHERE
			tab_113526.col_194263
		ORDER BY
			tab_113527.col_194264 ASC, tab_113527.col_194264
		LIMIT
			90:::INT8;
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlsmith/setup=empty/setting=no-mutations.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-18626

--- a/pkg/internal/sqlsmith/schema.go
+++ b/pkg/internal/sqlsmith/schema.go
@@ -517,6 +517,7 @@ var functions = func() map[tree.FunctionClass]map[oid.Oid][]function {
 			"crdb_internal.complete_replication_stream",
 			"crdb_internal.revalidate_unique_constraint",
 			"crdb_internal.request_statement_bundle",
+			"crdb_internal.set_compaction_concurrency",
 		} {
 			skip = skip || strings.Contains(def.Name, substr)
 		}

